firmware/.clang-format

```diff
@@ -12,3 +12,4 @@ IndentWidth: 4
 TabWidth: 4
 AccessModifierOffset: -4
 AllowShortFunctionsOnASingleLine: Empty
+QualifierAlignment: Right
```

Comment:
Let's add the wiki you linked me somewhere: https://isocpp.org/wiki/faq/const-correctness#const-ref-alt